.github/workflows/cmake.yml

```diff
@@ -6,28 +6,46 @@ on:
   pull_request:
     branches: [ master ]
 
-env:
-  CMAKE_TEST_PORJECT_PATH: build/tests
-  CMAKE_TEST_PORJECT_NAME: Template_CXX_tests
-
 jobs:
   build:
     runs-on: ubuntu-latest
+    container:
+      image: swarmus/github-ci-action
```

Comment:
Maybe swarmus/esp-idf or swarmus/embedded, something like that would be more accurate? Maybe eventually we will have an image for the C# repo for AR.  I know there's all the dependencies with ROS and all, i'm just note sure of the name. Where is the docker file btw?

Reply:
Switched to "embedded" image Charles made for ROS.